extensions/amp-analytics/0.1/events.js

```diff
@@ -46,6 +46,7 @@ const TAG = 'amp-analytics/events';
 export const AnalyticsEventType = {
   CLICK: 'click',
   CUSTOM: 'custom',
+  AMP_STORY: 'amp-story',
```

Comment:
I'm a bit confused by the two names. `amp-story` and `story`.  Any concern on using `story` instead?